src/panels/meeting-times.js

```diff
@@ -1,45 +1,64 @@
 var React = require('react');
+var $ = require('jquery');
 var classNames = require('classnames');
+var time = require('../utils/time.js');
+var moment = require('moment')
 
-var meetingTimes = require('../data/meeting-times.json');
+var secrets = require('../secrets.js');
+var groupsURL = secrets.grootServicesURL + '/groups/sigs'
 
 var ROWS_PER_PAGE = 9;
+var REFRESH_TIMES_MS = 60 * 1000;
+var SWITCH_PAGE_MS = 10 * 1000;
 
 /**
  * Meeting times panel.
  */
 var MeetingTimesPanel = React.createClass({
     getInitialState: function() {
         return {
-            index: 0
+            index: 0,
+            sigs: []
```

Comment:
Currently, just SIGs. Unfortunately, `groot-groups-service` doesn't have meeting times for committees.

i.e. the projects committee is just: 
```
  {
    "name": "Projects",
    "description": "Projects",
    "chairs": "Kevin Wang",
    "members": [
      "Kevin Wang"
    ],
    "netids": [
      "klwang3"
    ],
    "email": "Projects-l@acm.illinois.edu"
  }
```